dkg/pedersen/handler.go

```diff
@@ -123,8 +167,11 @@ func (h *Handler) Stream(out mino.Sender, in mino.Receiver) error {
 	}
 	h.Unlock()
 
-	deals := list.New()
-	responses := list.New()
+	deals := newCryChan[types.Deal](chanSize)
+	responses := newCryChan[*pedersen.Response](chanSize)
```

Comment:
is there a reason in one case we use `*pedersen.Response` and in the other `types.Deal` ?
i.e. in one case we process the message before adding it to the channel, in the other we add the raw network message to the channel ? 
i think that's a regression, we'd made this uniform :)

Reply:
I'm happy with both 🤷‍♂️

Reply:
I think there was a misunderstanding 🤦‍♂️ 

I have no issue with the pointer, but rather, where we receive the messages:
- in the response case, we convert them immediately (in `func Stream`, case `types.Response`) to a Kyber type ( `pedersen.Respones` )
- in the deal case, we store the `types.Deal` in `func Stream`, but then convert it to a Kyber type (`pedersen.Deal`) in `func handleDeal`.

Basically, it would make sense for the internal API to either be consistent about using `pedersen` types or `types` types 😄